NIPostsReload/Presentation/Scenes/NIPostFeedModule/Presenter/NIPostFeedPresenter.swift

```diff
@@ -7,6 +7,23 @@
 
 import UIKit
 
+enum PostFeedSortType: CaseIterable {
```

Comment:
If you need to declare only the title, consider using a raw value instead of a computed property